### PR TITLE
PDF viewer custom widget on FlutterFlow

### PR DESCRIPTION
**Pull request by immankumarsync**
## Description ##

* Added content to use PDF viewer widget on FlutterFlow

### CODE REVIEW

**Comment by immankumarsync** [comment]
rebuild

File: Flutter/pdf-viewer/How-to/custom-widget-on-flutterflow.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+---
+layout: post
```
Comment (Deepak1799) [inline Flutter/pdf-viewer/How-to/custom-widget-on-flutterflow.md:2]: For new pages, use keywords in the meta data

https://teams.microsoft.com/l/message/19:dd078bd9bab14ca98eca6e7f0b9293d1@thread.v2/1718013495233?context=%7B%22contextType%22%3A%22chat%22%7D

File: Flutter/pdf-viewer/How-to/custom-widget-on-flutterflow.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+title: PdfViewer Widget in FlutterFlow | Syncfusion
+description: You can learn here all about adding Syncfusion Flutter Pdf Viewer ( SfPdfViewer Widget ) widget in FlutterFlow.
```
Comment (Deepak1799) [inline Flutter/pdf-viewer/How-to/custom-widget-on-flutterflow.md:4]: There are spaces at the front and back inside the bracket content. Is this valid?

image

**Review by Deepak1799 — CHANGES_REQUESTED** [review]
Requested changes

File: Flutter/pdf-viewer/How-to/custom-widget-on-flutterflow.md (hunk 2 above)
Comment (immankumarsync) [reply Flutter/pdf-viewer/How-to/custom-widget-on-flutterflow.md]: Have removed the spaces

**Pull request merged**